lib/ext/heroku/command/addons.rb

```diff
@@ -341,25 +344,27 @@ def destroy
       ).body.keep_if do |attachment|
         attachment['addon']['name'] == addon
       end
-      config_vars = addon_attachments.map { |attachment| attachment['name'] }
 
-      config_vars.each do |var_name|
-        action("Removing #{addon} as #{var_name} from #{app}") {}
-        action("Unsetting #{var_name} and restarting #{app}") {}
+      attachment_names = addon_attachments.map { |attachment| attachment['name'] }
+      attachment_names.each do |attachment_name|
+        action("Removing #{addon} as #{attachment_name} from #{app}") {}
```

Comment:
I worry about abusing `action` here. It's going to say `... done` even if we haven't completed the action, because it doesn't happen until L353 below, and that can fail if there are attachments across multiple apps and the `--force` option isn't given.